sale_sourced_by_line/migrations/8.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent - Miquel Raich
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def populate_order_line_location_id(env):
+    env.cr.execute(
+        """
+        SELECT id, %s
+        FROM sale_order_line
+        """ % openupgrade.get_legacy_name('location_id')
+    )
+    locations = env.cr.fetchall()
```

Comment:
Why locations instead of so_lines?

Reply:
It's just a name. That fetch gives a list of two fields: one is the so_line.id and the other is the so_line.location_id.

Reply:
You're right. It's ok to me.